indico/htdocs/sass/modules/_papers.scss

```diff
@@ -63,3 +63,114 @@
         width: 4rem;
     }
 }
+
+.display-contribution-list {
+    @extend %font-family-modern-body;
+
+    .contribution-row {
+        @include border-left($width: 3px);
+        padding-left: 10px;
+        margin-bottom: 1em;
+
+        &:hover {
+            border-color: $blue;
+        }
+
+        .row-title {
+            @include border-bottom();
+            font-size: 14pt;
+        }
+
+        .submitter {
+            color : $light-black;
+            margin-top: 7px;
+        }
+
+        .label {
+            border: 0;
```

Comment:
Our sass-list says that we use `border: none;` https://github.com/indico/indico/blob/devel/.sass-lint.yml#L13